### DISCUSSION TITLE
#GitHubCopilot & #JetBrainsFleet - Made for Each Other as #PairProg platform

### DISCUSSION THREAD

**Discussion by Jim-Salmons** [community/community#8758] [Copilot Conversations]
Heads up @ryanjsalva & #Copilot DevTeam - As a #DisabledDeveloper, I have great respect and interest in how Copilot can empower both individual and Pair Programmers [See my [Copilot for Disabled Developers publication](https://medium.com/github-copilot-for-disabled-developers).].

This week I happened upon a link to a [video demo](https://youtu.be/ow5kdhDa_pk) of the upcoming next generation IDE, **[Fleet](https://www.jetbrains.com/fleet/)**, from JetBrains. Sadly, the Preview program is closed ATM. I sent them an email to see if I can get into the Fleet Preview based on my long-time customer relationship during my working career and their current support of my #DigitalHumanities #CitizenScientist research by way of the JetBrains Open Source Developer Program. 

Upon watching the demo, I immediately saw the incredible potential that a **Copilot-enhanced Fleet** could be. This would be awesome not just for disabled developers but for all devs, especially those doing pair- and small-team development. I hope you folks are already collaborating on this integration.  If not, I hope this post and my email suggesting this to JetBrains helps to stimulate collaboration to this end.

BTW, @ryanjsalva, although I have been quiet here and via email to you about co-developing a **Copilot for Disabled Developers support program and research study**, this does not indicate a waning of my interest. I have been in exploratory contact with a number of individuals and organizations to gauge interest in support and participation. An update to follow.

Happy Healthy Vibes from Colorado,
  Jim

**Comment by SmallLemur** [comment]
Hi Jim! This is very exciting and we are really just beginning to consider the possibility of the future development tool !
thank you for the great feedback. I would be very interested in having a discussion with you regarding this topic and also hearing a bit more your feedback. Would you be interested for an interview of 30min ?

**Comment by PeterCullenBurbery** [comment]
The Fleet preview is now available at https://www.jetbrains.com/fleet/
